ubuntu-ppc64le-cuda/8.0-cudnn6/Dockerfile

```diff
@@ -0,0 +1,99 @@
+FROM nvidia/cuda-ppc64le:8.0-cudnn6-devel-ubuntu16.04
+LABEL MAINTAINER="OSU Open Source Lab <dockerhub@osuosl.org>"
+
+# Below was taken from https://github.com/jenkinsci/docker-ssh-slave
+
+ARG user=jenkins
+ARG group=jenkins
+ARG uid=1000
+ARG gid=1000
+ARG JENKINS_AGENT_HOME=/home/${user}
+
+ENV JENKINS_AGENT_HOME ${JENKINS_AGENT_HOME}
+ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk
+ENV PATH $PATH:$JAVA_HOME/bin
+
+RUN groupadd -g ${gid} ${group} && \
+    useradd -d "${JENKINS_AGENT_HOME}" -u "${uid}" -g "${gid}" -m -s /bin/bash "${user}"
+
+# install Java
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y openjdk-8-jdk && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# install git
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y git && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# setup sudo
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y sudo && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* && \
```

Comment:
why is `git`, `openjdk-8-jdk`, and `sudo` installed separately? Couldn't they all be included in the same `RUN` command?

Reply:
They could but I'm trying to separate out what each section is doing.

Reply:
For sudo it makes sense, since you are configuring the package at the same time as installation. but for git and openjdk-8-jdk, they don't require any extra configuration, and could be installed with the giant group of packages later in the Dockerfile

Reply:
Ya I suppose it might be better to just lump it together.